test/accumulators_fraction_test.cpp

```diff
@@ -28,6 +28,23 @@ void run_tests() {
     BOOST_TEST_EQ(f.failures(), 0);
   }
 
+  {
+    f_t f;
+    f(true); f(true); f(true);
+    f(false); f(false);
+    BOOST_TEST_EQ(f.successes(), 3);
+    BOOST_TEST_EQ(f.failures(), 2);
+  }
+
+  {
+    using f_t1 = fraction<double>;
+    using f_t2 = fraction<int>;
+    f_t1 f1(5,3);
+    f_t2 f(f1.successes(), f1.failures());
+    BOOST_TEST_EQ(f.successes(), 5);
+    BOOST_TEST_EQ(f.failures(), 3);
+  }
```

Comment:
that's not how the converting constructor works. What you are testing here always works, even without making a special constructor. You need to test `f_t1 f1(5, 3); f_t2 f2(f1);`

Reply:
The other test you added looks good. :+1: